docs/source-pytorch/clouds/cluster_intermediate_2.rst

```diff
@@ -35,3 +35,13 @@ Run the below command with the appropriate variables set on each node.
 .. note::
 
     ``torch.distributed.run`` assumes that you'd like to spawn a process per GPU if GPU devices are found on the node. This can be adjusted with ``-nproc_per_node``.
```

Comment:
While we are here, could we fix this typo too? :))
```suggestion
    ``torch.distributed.run`` assumes that you'd like to spawn a process per GPU if GPU devices are found on the node. This can be adjusted with ``--nproc_per_node``.
```

Reply:
btw, your new PR is planning to close this PR, so this fix will get lost.

Reply:
We can keep it open and land any potential improvements to the docs we can come up with. If #18618 lands, the info below technically wouldn't be necessary anymore since it will just work.